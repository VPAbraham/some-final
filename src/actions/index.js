export const setOrders = orders => ({
  type: 'SET_ORDERS',
  orders
});

export const addOrders = orders => ({
  type: 'ADD_ORDERS',
  orders
})
