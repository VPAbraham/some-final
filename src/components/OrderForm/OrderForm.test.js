import React from 'react';
import { OrderForm, mapStateToProps, mapDispatchToProps } from './OrderForm';
import { shallow } from 'enzyme';
import { setOrders, addOrders } from '../../actions';

describe('OrderForm', () => {
  let wrapper;

  const mockOrders = [{ id: 1, name: 'Robbie', ingredients: ['stuff1', 'stuff2'] },
  { id: 2, name: 'Travis', ingredients: ['stuff3', 'stuff4'] }];

  beforeEach(() => {
    wrapper = shallow(<OrderForm
      orders={mockOrders}
      setOrders={jest.fn()}
      addOrders={jest.fn()}
    />)

  });

  it('should match the snapshot with the data passed in', () => {
    expect(wrapper).toMatchSnapshot()
  })
});