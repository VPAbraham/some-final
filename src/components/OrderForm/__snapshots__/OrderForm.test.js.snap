// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrderForm should match the snapshot with the data passed in 1`] = `
<form>
  <input
    name="name"
    onChange={[Function]}
    placeholder="Name"
    type="text"
    value=""
  />
  <button
    className="beans"
    key="beans"
    name="beans"
    onClick={[Function]}
  >
    beans
  </button>
  <button
    className="steak"
    key="steak"
    name="steak"
    onClick={[Function]}
  >
    steak
  </button>
  <button
    className="carnitas"
    key="carnitas"
    name="carnitas"
    onClick={[Function]}
  >
    carnitas
  </button>
  <button
    className="sofritas"
    key="sofritas"
    name="sofritas"
    onClick={[Function]}
  >
    sofritas
  </button>
  <button
    className="lettuce"
    key="lettuce"
    name="lettuce"
    onClick={[Function]}
  >
    lettuce
  </button>
  <button
    className="queso fresco"
    key="queso fresco"
    name="queso fresco"
    onClick={[Function]}
  >
    queso fresco
  </button>
  <button
    className="pico de gallo"
    key="pico de gallo"
    name="pico de gallo"
    onClick={[Function]}
  >
    pico de gallo
  </button>
  <button
    className="hot sauce"
    key="hot sauce"
    name="hot sauce"
    onClick={[Function]}
  >
    hot sauce
  </button>
  <button
    className="guacamole"
    key="guacamole"
    name="guacamole"
    onClick={[Function]}
  >
    guacamole
  </button>
  <button
    className="jalapenos"
    key="jalapenos"
    name="jalapenos"
    onClick={[Function]}
  >
    jalapenos
  </button>
  <button
    className="cilantro"
    key="cilantro"
    name="cilantro"
    onClick={[Function]}
  >
    cilantro
  </button>
  <button
    className="sour cream"
    key="sour cream"
    name="sour cream"
    onClick={[Function]}
  >
    sour cream
  </button>
  <p>
    Order: 
    Nothing selected
  </p>
  <button
    className="sub-btn"
    onClick={[Function]}
  >
    Submit Order
  </button>
</form>
`;
